docs/astro/src/content/docs/guide/platforms/mobile.mdx

```diff
@@ -9,11 +9,13 @@ we recommend using a recent desktop operating system and compiler version.
 
 ## Android
 
-Slint runs on Android. See [GitHub Tracking Issue](https://github.com/slint-ui/slint/issues/46)
+Slint on Android is currently only supported out of the box for the Rust programming language.
```

Comment:
```suggestion
Slint on Android is only supported for the Rust programming language.
```

Reply:
You removed "currently", meaning we should not give hope to users that it will work with other programming languages?

Reply:
I think documentation should be precise and concise.

“not currently supported” is vague due to the lack of conditions, it may create false expectations about this being a temporary situation - especially since from our side there is not concrete plan or issue we can refer to. (“Not currently supported due to limitation XYZ that will be resolved in the next release”, like a known issue to be fixed)

Generally, the term “currently” is time relative and that’s not great for documentation as its meaning changes over time while the documentation doesn’t. We release often, so it’s not typical right now that somebody reads docs for say the 1.0.3 release where the docs may say “currently not supported”, but I’d rather we avoid a habit that would bite us later when it will happen that somebody uses Slint LTS 2.0.1 at the time when the latest release is say 2.10. In such a context the time relative “currently” is not a good choice IMO.